### ISSUE TITLE
Suggestion: setlogin timer

### ISSUE THREAD

**Issue by AvalancheYT** [TotalFreedom/TotalFreedomMod#1631]
I was thinking that we should add the timer so when an admin set a login message, he won't able to change it for another 30 days. Similar to our old system...

**Comment by DUFCLiam** [comment]
I do not like this as if they make a mistake. They cannot fix it for 30 days.

**Comment by AvalancheYT** [comment]
We can always add confirm part.

**Comment by DUFCLiam** [comment]
Then due to that I agree! It will say:
`
Type /myadmin setlogin confirm
`

**Comment by Cleinad119** [comment]
I agree. However, there should be an option to disable/enable the timer, and how long.
